sdk/src/main/java/com/horizen/account/transaction/EthereumTransactionSerializer.java

```diff
@@ -0,0 +1,47 @@
+package com.horizen.account.transaction;
+
+import com.horizen.account.utils.EthereumTransactionDecoder;
+import com.horizen.transaction.TransactionSerializer;
+import org.web3j.crypto.*;
+import org.web3j.utils.Numeric;
+import scorex.util.serialization.Reader;
+import scorex.util.serialization.Writer;
+
+public class EthereumTransactionSerializer implements TransactionSerializer<EthereumTransaction> {
+
+    private static final EthereumTransactionSerializer serializer;
+
+    static {
+        serializer = new EthereumTransactionSerializer();
+    }
+
+    private EthereumTransactionSerializer() {
+        super();
+    }
+
+    public static EthereumTransactionSerializer getSerializer() {
+        return serializer;
+    }
+
+    // Maybe we need to do the serialization in a different way to be eth compatible,
```

Comment:
We are not eth compatible here, but to me it seems there is no way of avoiding using the length of the byte array.
Moreover this de/serialization framework is internal to the sdk only, for instance, in the `eth_sendRawTransaction` rpc cmd we are not using this decoding but the one eth compliant (see: `sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala`)